flowkit/config/json/dependency_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package json
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ConfigDependencies(t *testing.T) {
+	b := []byte(`{
+		"HelloWorld": "testnet://877931736ee77cff.HelloWorld"
+		}`)
+
+	var jsonDependencies jsonDependencies
+	err := json.Unmarshal(b, &jsonDependencies)
+	assert.NoError(t, err)
+
+	dependencies, err := jsonDependencies.transformToConfig()
+	assert.NoError(t, err)
+
+	assert.Len(t, dependencies, 1)
+
+	dependencyOne := dependencies.ByName("HelloWorld")
+	assert.NotNil(t, dependencyOne)
+
+	assert.NotNil(t, dependencyOne)
+}
+
+func Test_TransformDependenciesToJSON(t *testing.T) {
+	b := []byte(`{
+		"HelloWorld": "testnet://877931736ee77cff.HelloWorld"
```

Comment:
This comment applies to a lot more than just this line. I think it's best to always write addresses with `0x` prefix because they are more clear to read. This applies to folders we create under `imports` as well. Anywhere in the new JSON structure when an address is written (like in simple format) it should have `0x` prefix. What do you think?

Reply:
I suggest using identifier here because it is known already. So A.address.contract

Reply:
I like `0x` but it may cause some inconsistencies with other pieces of the CLI. For example, the addresses created on `init` are this format, as well as addresses created by `flow dev`.